go-controller/pkg/ovn/obj_retry.go

```diff
@@ -833,6 +850,11 @@ func (oc *Controller) iterateRetryResources(r *retryObjs, updateAll bool) {
 	for objKey, entry := range r.entries {
 		if entry.ignore {
 			continue
+		} else if entry.failedRetries >= maxRetries {
+			klog.Warningf("Dropping retry entry for %s %s: exceeded number of retries (%d)",
```

Comment:
not sure u be able to exceed the threshold so not sure if there is a value printing it

Reply:
yeah, let me remove the `%d`